File: src/main/webapp/app/exercises/shared/plagiarism/types/modeling/ModelingPlagiarismResult.ts
import { ModelingComparison } from './ModelingComparison';
import { PlagiarismResult } from '../PlagiarismResult';

/**
 * Result of the automatic plagiarism detection for modeling exercises.
 */
export class ModelingPlagiarismResult extends PlagiarismResult<ModelingComparison> {
    /**
     * Currently, the `ModelingPlagiarismResult` does not have any meta information and only contains
     * a list of comparisons inherited from `PlagiarismResult`.
     */
}

File: src/main/java/de/tum/in/www1/artemis/domain/plagiarism/modeling/ModelingPlagiarismResult.java
package de.tum.in.www1.artemis.domain.plagiarism.modeling;

import de.tum.in.www1.artemis.domain.plagiarism.PlagiarismResult;
import jakarta.persistence.Entity;

/**
 * Result of the automatic plagiarism detection for modeling exercises.
 */
@Entity
public class ModelingPlagiarismResult extends PlagiarismResult<ModelingSubmissionElement> {

}
